src/collections/Table/Table.js

```diff
@@ -10,43 +10,97 @@ export default class Table extends Component {
     children: ofComponentTypes(['TableColumn']),
     className: PropTypes.string,
     data: PropTypes.array,
+    onSelectRow: PropTypes.func,
     onSortChange: PropTypes.func,
+    selectedRows: PropTypes.arrayOf(PropTypes.number),
     sort: PropTypes.shape({
       key: PropTypes.string,
-      direction: PropTypes.string, // descending|ascending, defaults to descending
+      direction: PropTypes.oneOf(['descending', 'ascending']),
     }),
   };
 
+  static defaultProps = {
+    selectedRows: [],
+    sort: {
+      key: null,
+      direction: 'descending',
+    },
+  }
```

Comment:
Also added sort props here, removes need for more complicated `_.get()` use later.